live-examples/css-examples/text-decoration/text-emphasis.html

```diff
@@ -14,7 +14,7 @@
     </div>
 
     <div class="example-choice">
-        <pre><code class="language-css">text-emphasis: 'x';</code></pre>
+        <pre><code class="language-css">text-emphasis: <code style="color: #d8ff00">'x'</code>;</code></pre>
```

Comment:
This shouldn't be changed. The colors of CSS declarations are controlled by BOB.